letsfarm-apps/letsfarm-api
# Change to sequelize

### what does this Pr do?
changes the application to use Sequelize

### what tasks does it accomplish?
- creation of migrations and models for the whole application
- adding relationship mappings 
- modify controllers to work with on new models
- remove old files for the database, custom models, query builder 

### How should this task be tested?
- switch to this branch
- run `npm install` to install dependencies 
- run `sequelise db:migrate` to create the database tables from migrations 
NB. ensure that sequelise is installed globally for the `sequelise` comand to run
